### DISCUSSION TITLE
13.0.1 Performance bad after configuration complete

### DISCUSSION THREAD

**Discussion by benbecke** [jomjol/AI-on-the-edge-device#1442] [Q&A]
Made a fresh install of 13.0.1. 
After configuring reference picture , marks and digits and rebooting multiple times performance gets bad ...
Board is even not reacting well 

![image](https://user-images.githubusercontent.com/56760381/204893323-dbfcb20e-39b1-48e1-b4fb-0717b122ac41.png)

Log is telling:

2022-11-30T15:18:12 <INF> [MAIN] =============================================================================================
2022-11-30T15:18:12 <INF> [MAIN] Branch: 'HEAD', Revision: b60c2d6, Date/Time: 2022-11-30 09:26, Web UI: Development-Branch: HEAD (Commit: b60c2d6)
2022-11-30T15:18:12 <INF> [MAIN] Reset reason: Exception/panic
2022-11-30T15:18:15 <INF> [MAIN] Initialization completed successfully!
**2022-11-30T15:18:15 <WRN> [TFLITE] Restarted due to an Exception/panic! Postponing first round start by 5 minutes to allow for an OTA or to fetch the log!**
2022-11-30T15:18:15 <WRN> [TFLITE] Setting logfile level to DEBUG until the next reboot!
2022-11-30T15:18:26 <DBG> [MAIN SERVER] info_get_handler
2022-11-30T15:18:26 <DBG> [MAIN SERVER] info_get_handler
2022-11-30T15:18:26 <DBG> [MAIN SERVER] info_get_handler
2022-11-30T15:18:57 <DBG> [MAIN SERVER] info_get_handler
2022-11-30T15:18:57 <DBG> [MAIN SERVER] info_get_handler
2022-11-30T15:18:59 <DBG> [MAIN SERVER] info_get_handler
2022-11-30T15:19:01 <DBG> [MAIN SERVER] info_get_handler
2022-11-30T15:19:02 <DBG> [MAIN SERVER] info_get_handler

My config :

[MakeImage]
;LogImageLocation = /log/source
WaitBeforeTakingPicture = 5
;LogfileRetentionInDays = 15
Brightness = 0
Contrast = 1
Saturation = 0
LEDIntensity = 35
ImageQuality = 12
ImageSize = VGA
FixedExposure = false

[Alignment]
InitialRotate = 95
InitialMirror = false
SearchFieldX = 20
SearchFieldY = 20
AlignmentAlgo = default
FlipImageSize = false
/config/ref0.jpg 382 76
/config/ref1.jpg 489 147

[Digits]
Model = /config/dig-cont_0600_s3.tflite
CNNGoodThreshold = 0.5
;LogImageLocation = /log/digit
;LogfileRetentionInDays = 3
main.dig1 264 124 34 62 false
main.dig2 305 124 34 62 false
main.dig3 345 124 34 62 false
main.dig4 382 124 34 62 0
main.dig5 417 124 34 62 0

;[Analog]
Model = /config/ana-cont_11.3.0_s2.tflite
;LogImageLocation = /log/analog
;LogfileRetentionInDays = 3
main.ana1 432 230 92 92 false
main.ana2 379 332 92 92 false
main.ana3 283 374 92 92 false
main.ana4 155 328 92 92 false

[PostProcessing]
main.DecimalShift = -2
main.AnalogDigitalTransitionStart = 9.2
PreValueUse = true
PreValueAgeStartup = 720
AllowNegativeRates = false
main.MaxRateValue = 1
;main.MaxRateType = AbsoluteChange
;main.ExtendedResolution = false
;main.IgnoreLeadingNaN = true
ErrorMessage = true
CheckDigitIncreaseConsistency = false

;[MQTT]
;Uri = mqtt://IP-ADRESS:1883
;MainTopic = wasserzaehler
;ClientID = wasser
;user = USERNAME
;password = PASSWORD
;SetRetainFlag = true
;HomeassistantDiscovery = 
;MeterType = 

;[InfluxDB]
;Uri = undefined
;Database = 
;Measurement = undefined
;user = undefined
;password = undefined

[GPIO]
;IO0 = input disabled 10 false false 
;IO1 = input disabled 10 false false 
;IO3 = input disabled 10 false false 
IO4 = built-in-led disabled 10 false false 
;IO12 = external-flash-ws281x disabled 10 false false 
;IO13 = input-pullup disabled 10 false false 
LEDType = WS2812
LEDNumbers = 5
LEDColor = 150 150 150

[AutoTimer]
AutoStart = true
Intervall = 5

[DataLogging]
DataLogActive = false
DataLogRetentionInDays = 3

[Debug]
Logfile = 1
LogfileRetentionInDays = 3

[System]
TimeZone = CET-1CEST,M3.5.0,M10.5.0/3
;TimeServer = undefined
;AutoAdjustSummertime = false
;Hostname = undefined
SetupMode = false

12.0.1 ran very well.....

**Comment by caco3** [comment]
Please use latest rolling ans show us logs covering the restart.
You can get it from here: https://github.com/jomjol/AI-on-the-edge-device/actions/workflows/build.yaml?query=branch%3Arolling

**Reply by caco3** [reply]
how us the latest release working for you?

**Comment by benbecke** [comment]
Sorry didn‘t have time . Think will check next week
